routing/graph.go

```diff
@@ -20,48 +20,48 @@ type routingGraph interface {
 	fetchNodeFeatures(nodePub route.Vertex) (*lnwire.FeatureVector, error)
 }
 
-// dbRoutingTx is a routingGraph implementation that retrieves from the
+// CachedGraph is a Graph implementation that retrieves from the
```

Comment:
nit: could add compile time check here to ensure `CachedGraph` implements the `Graph` interface.